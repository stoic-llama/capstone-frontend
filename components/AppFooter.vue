<template>
    <footer class="container-fluid d-flex align-items-center justify-content-center p-2 bg-warning text-center">
        © 2022-{{ currentYear }} // Designed & Coded with 💙 by Parents and Community Volunteers 🍼
    </footer>
</template>

<script>
export default {
    name: 'AppFooter',
    computed: {
        currentYear() {
            return new Date().getFullYear().toString()            
        }
    }
}

</script>

<style lang="scss" scoped>

</style>