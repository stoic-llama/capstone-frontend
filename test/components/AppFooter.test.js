import { mount } from '@vue/test-utils'
import AppFooter from '@/components/AppFooter.vue'

test('mount AppFooter component', () => {
    const wrapper = mount(AppFooter)

    expect(wrapper.html()).toMatchSnapshot()
})


