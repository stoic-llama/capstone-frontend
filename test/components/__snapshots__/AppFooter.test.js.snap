// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mount AppFooter component 1`] = `
"<footer class="container-fluid d-flex align-items-center justify-content-center p-2 bg-warning text-center">
  © 2022-2025 // Designed &amp; Coded with 💙 by Parents and Community Volunteers 🍼
</footer>"
`;
